input/includes/observation_vitalsigns_parameters.md

```diff
@@ -6,6 +6,12 @@
     <th>Type(s)</th>
     <th>Requirements (when used alone or in combination)</th>
   </tr>
+  <tr>
+        <td>patient</td>
+        <td><b>SHALL</b></td>
```

Comment:
As above - Conformance changed from MAY to SHALL. Previously, patient was MAY and patient.identifier was SHOULD. CapabilityStatements are MAY for patient, including chained search patient.identifer. Please clarify the change.